src/Misc/externals.sh

```diff
@@ -3,6 +3,7 @@ PACKAGERUNTIME=$1
 PRECACHE=$2
 
 NODE_URL=https://nodejs.org/dist
+NODE_UNOFFICIAL_URL=https://unofficial-builds.nodejs.org/download/release
```

Comment:
what is unoffical-builds?  is that trusted?

Reply:
The NodeJS can't currently publish official builds for Windows arm64 yet due to missing Hosted ARM64 runners (tracked in https://github.com/nodejs/build/issues/2450#issuecomment-705853342). Because of this, the NodeJS Build team is currently providing unofficial builds at that URL.

Compilation support for ARM64 [is at Tier 2](https://github.com/nodejs/node/blob/master/BUILDING.md#platform-list), meaning that they won't release new versions if NodeJS can't be built on arm64.

Reply:
Would the existing node we package work on the windows arm64 through an emulation layer?

Reply:
@bryanmacfarlane it would indeed, however it would result in a loss of performance as `process.arch` would also return `x64` in that case. We should [actually be very close to official arm64 builds](https://github.com/nodejs/build/issues/2918#issuecomment-1156497518), so may I suggest that we wait 2-3 more weeks? Otherwise I'm happy to update this PR to use `x64` Node until the native version becomes available.